### ISSUE TITLE
"Value for the fixme tag if needed:" should stay

### ISSUE THREAD

**Issue by sfkeller** [mapsme/cf_audit#20]
Often the same value needs to be entered. So, the optional field for #fixme ("Value for the fixme tag if needed:") should stay, even after saving an update.

**Comment by Zverik** [comment]
Users can easily forget to clear this field, and since the next point is chosen by random, it will seldom apply to it.

**Comment by sfkeller** [comment]
Agreed. I did non mean a default value, but just enabling autocomplete. This is typically supported by the browsers inside a form (like this "<form action="..." autocomplete="on">),
